components/auth_modal/view.coffee

```diff
@@ -32,7 +32,7 @@ module.exports = class AuthModalView extends ModalView
   initialize: (options) ->
     return if isEigen.checkWith options
 
-    { @destination } = options
+    { @destination, @successCallback } = options
```

Comment:
Does this get used anywhere?

Reply:
I’m guessing not. I came here to ask the same thing 😂